### DISCUSSION TITLE
Want to read calendar of an outlook.com account (free email and so on)

### DISCUSSION THREAD

**Discussion by ghost** [O365/python-o365#971] [General]
Greetings,

The museum I work for wants to use  a free Outlook.com account for multiple calendars. That's fine. But I would like to be able to connect from my Python application to that account and (at least) read calendar events. I have experience with (non free) Azure Outlook calendars for this organisation so also some basic knowledge of these things. 

And now I want to connect to the (free) outlook.com account and did read this scenario: 

https://learn.microsoft.com/en-us/azure/active-directory/develop/scenario-daemon-overview

Is this correct way of working or should I consider another scenario to start with?

**Comment by alejcas** [comment]
Sorry I can't guide you on which scenario is better or worse
